spec/unit/containerize_task_spec.rb

```diff
@@ -56,5 +57,16 @@
         to receive(:write_compose_file).with(workloads, output_path)
       containerize_task.containerize(system_description, output_path)
     end
+
+    it "shows the number of detected workloads" do
+      expect_any_instance_of(WorkloadMapper).
+        to receive(:identify_workloads).with(system_description).and_return(workloads)
+      expect_any_instance_of(WorkloadMapper).
+        to receive(:write_compose_file).with(workloads, output_path)
+      containerize_task.containerize(system_description, output_path)
+      expect(captured_machinery_output).to include(
```

Comment:
I would not check that the string is included, but compare against the full output.

Specifying the output with the heredoc syntax makes it easier to read, because it than looks exactly like it will look like to the user.